### DISCUSSION TITLE
PyFluent feedback

### DISCUSSION THREAD

**Discussion by PyPablo** [ansys/pyfluent#69] [Q&A] [answered]
I've been playing around with the tool and so far so good. It really works like a charm. For the little I've navigated through, I've seen
1) the connectivity to the attach session is very robust. I only managed to break it when passing weird formatted strings to the Scheme interpreter.
 
2) Documentation is awesome as well as the intellisense. This is definitely going to make life much easier to customers.
3) The tree-based architecture works great. I see it is not a 1-to-1 version of the tree as of yet (Calculation activities, cell registers, methods, etc.). Are we planning to add these as well?

**Comment by h-krishnan** [comment, marked as answer]
We are working on expanding the tree-based architecture to cover the entire breadth of Fluent functionality.
